packages/app-store/routing-forms/pages/form-edit/[...appPages].tsx

```diff
@@ -213,6 +237,7 @@ const FormEdit = ({
   } = useFieldArray({
     control: hookForm.control,
     name: fieldsNamespace,
+    keyName: "_id",
```

Comment:
The default value for `keyName` is `id`. Although we already have `field.id` value (from our database), React Hook Form overrides it with their autogenerated value.

![Screenshot 2025-06-25 at 14 10 25](https://github.com/user-attachments/assets/9ff7d316-c251-4e7a-97ee-d50649f4b96f)

Luckily, when we update the field, we use the array index. So it hasn't been a problem. But because we want to compare if a field is newly added or already in the database, we need to fix this.